config/environments/development.rb

```diff
@@ -18,7 +18,7 @@
 
   # Don't care if the mailer can't send.
   config.action_mailer.raise_delivery_errors = true
-  config.action_mailer.delivery_method = :letter_opener
+  config.action_mailer.delivery_method = :smtp
```

Comment:
This should be `letter_opener` in development. When we deploy to staging will use `smtp`.